content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -436,7 +491,6 @@ can be found under :menuselection:`Accounting --> Reporting --> Colombian Statem
 de Retención en IVA`.
 
 .. image:: colombia/iva-report.png
```

Comment:
@samueljlieber Change to `retencion-iva-dian.png` already in [Google Drive folder](https://drive.google.com/drive/folders/11cXF1gXxwws_P9hUslnSJcfjheEbBJZN) :)